puppet/modules/jenkins_job_builder/files/theforeman.org/scripts/test/test_develop.sh

```diff
@@ -33,7 +40,7 @@ bundle exec rake rubocop
 ) > $APP_ROOT/config/database.yml
 
 # we need to install node modules for integration tests (which only run on postgresql)
-if [ ${database} = postgresql -a -e "$APP_ROOT/package.json" ]; then
+if [ ${UI} -eq true ]; then
```

Comment:
I suspect this will break when it's unset:
```
$ if [ ${UNSET_VAR} -eq true ] ; then echo test; fi
bash: [: -eq: unary operator expected
```

Reply:
@ekohl thanks, updated all conditions